app/models/concerns/versioning.rb

```diff
@@ -41,7 +41,8 @@ def save_version
   end
 
   def latest_version_id
-    send(self.class.versioned_association).order(id: :desc).select(:id).first.id
+    latest_version = send(self.class.versioned_association).order(id: :desc).select(:id).first
```

Comment:
Will this just stick nil into the db when it's expecting a version id value?
https://github.com/zooniverse/Panoptes/blob/7c8b9f28b4a66406e196c7a3ab1548eb7bd2dd7e/app/models/translation.rb#L28

Reply:
Yes. It might be a little more work for the frontend to take it into account. The flip-side is that using e.g. `0` instead of `nil` will mean that you can't use whatever number there to look up the actual Version object. Even though we have no plans to do so now, it seemed like a worse option.

Reply:
So the translation `string_versions` attribute will be missing values for these particular non-backfilled models. Should we just backfill the data to avoid this inconsistency with the new schema?

Reply:
Now the data is backfilled, what's the status of this?